packages/frontend/src/translations/en.global.json

```diff
@@ -1439,6 +1439,16 @@
             "resending": "Sending...",
             "resent": "Code sent!",
             "title": "Enter Two-Factor Verification Code"
+        },
+        "action": {
+            "addKey": {
+                "limited": "Adding key ${publicKey} limited to call ${methodNames} on ${receiverId} and spend up to ${allowance} on gas",
+                "full": "WARNING: Entering the code below will authorize full access to your NEAR account: ${accountId}. If you did not initiate this action, please DO NOT continue.<br /><br />This should only be done if you are adding a new seed phrase to your account. In all other cases, this is very dangerous.<br /><br /> The public key you are adding is: ${publicKey} ",
+            },
+            "deleteKey": "Deleting key ${publicKey}",
+            "functionCall": "Calling method: ${methodName} in contract: ${receiverId} with amount ${deposit} and with args ${args}",
+            "stake": "Staking: ${deposit} to validator: ${receiverId}",
+            "transfer": "Transferring: ${deposit}Ⓝ to ${receiverId}"
```

Comment:
nit: whenever you see the `Ⓝ` symbol in legacy copy for NEAR currency, let's switch it out for just `NEAR` instead :) We used to have inconsistent usage of the currency symbol but have settled on just using `NEAR`

Reply:
Sounds good! 👍🏽